torch/functional.py

```diff
@@ -245,3 +246,23 @@ def maybeSqueeze(tensor):
 
     raise ValueError("both arguments to __matmul__ need to be at least 1D, "
                      "but they are {}D and {}D".format(dim_tensor1, dim_tensor2))
+
+
+def sum(input, axes, keepdims=False, out=None):
+    if isinstance(axes, collections.Iterable)
+        if a.dim() > 3:
+            if keepdims:
```

Comment:
Do we need those two conditions? Can't we just pass the `keepdims` argument, and iterate with `sorted`?